sdk/tables/data-tables/src/TableClient.ts

```diff
@@ -549,25 +550,43 @@ export class TableClient {
     }
   }
 
-  /**
-   * Creates a new Batch to collect sub-operations that can be submitted together via submitBatch
-   * @param partitionKey - partitionKey to which the batch operations will be targetted to
-   */
-  // eslint-disable-next-line @azure/azure-sdk/ts-naming-options
-  public createBatch(partitionKey: string): TableBatch {
-    const batchId = Uuid.generateUuid();
+  public async submitTransaction(actions: TransactionAction[]): Promise<TableTransactionResponse> {
```

Comment:
In the new doc comment for this method, we should document the `TableTransaction` helper and maybe show a code snippet of using it